src/validation/extensions/maxar/MaxarValidatorCommon.ts

```diff
@@ -0,0 +1,246 @@
+import { defined } from "3d-tiles-tools";
+
+import { ValidationContext } from "../../ValidationContext";
+import { BasicValidator } from "../../BasicValidator";
+
+import { JsonValidationIssues } from "../../../issues/JsonValidationIssues";
+
+/**
+ * A class for validating common elements that appear in both
+ * `MAXAR_image_ortho` and `MAXAR_grid` extensions.
+ *
+ * @internal
+ */
+export class MaxarValidatorCommon {
+  /**
+   * Validates the spatial reference system (srs) object
+   *
+   * @param path - The path for ValidationIssue instances
+   * @param srs - The srs object to validate
+   * @param context - The ValidationContext that any issues will be added to
+   * @returns Whether the object was valid
+   */
+  static validateSrs(
+    path: string,
+    srs: any,
+    context: ValidationContext
+  ): boolean {
+    // Make sure that the given value is an object
+    if (!BasicValidator.validateObject(path, "srs", srs, context)) {
+      return false;
+    }
+
+    let result = true;
+
+    // Validate required properties
```

Comment:
This is first checking whether all required properties are present. Then (below) it is checking each required property, but only if it is `defined(...)`.

I think that _this_ check here could be omitted. And instead, the checks below could all do their checks without the `if (defined(...))` wrapped around them.

(This should make no difference in behavior, but maybe it could be a bit simpler or clearer)

Reply:
I have removed the `if (defined(...))` checks for each property below. But I still think that this test makes the validator a bit more readable IMHO.